### ISSUE TITLE
train_MambaDFuse训练卡在加载预训练模型

### ISSUE THREAD

**Issue by 1354555126** [Lizhe1228/MambaDFuse#22]
Model/Infrared_Visible_Fusion\Infrared_Visible_Fusion
Model/Infrared_Visible_Fusion\Infrared_Visible_Fusion
Model/Infrared_Visible_Fusion\Infrared_Visible_Fusion\options
Model/Infrared_Visible_Fusion\Infrared_Visible_Fusion\models
Model/Infrared_Visible_Fusion\Infrared_Visible_Fusion\images
LogHandlers setup!
Random seed: 563
Dataset: MEF for Multi-exposure Image Fusion.
Dataset [Dataset - train_dataset] is created.
Dataset: MEF for Multi-exposure Image Fusion.
Dataset [Dataset - test_dataset] is created.
in_chans:  1
Pass this initialization! Initialization was done during network defination!
in_chans:  1
Pass this initialization! Initialization was done during network defination!
Training model [ModelPlain] is created.
Loading model for G [Model/Infrared_Visible_Fusion\Infrared_Visible_Fusion\models\10000_G.pth] ...
Loading model for E [Model/Infrared_Visible_Fusion\Infrared_Visible_Fusion\models\10000_E.pth] ...
Loading optimizerG [Model/Infrared_Visible_Fusion\Infrared_Visible_Fusion\models\10000_optimizerG.pth] ...

**Comment by sucessfullys** [comment]
请问你解决了嘛

**Comment by sql58** [comment]
请问解决了吗
